includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -459,6 +463,10 @@ private function getVerticalData() {
 		$wikiFactoryInstance = WikiFactoryHub::getInstance();
 		$verticalData = $wikiFactoryInstance->getWikiVertical( $this->wikiId );
 
+		/**
+		 * We don't want to show vertical 'Other' instead we show vertical 'Lifestyle'
+		 * This is Comscore requirement
```

Comment:
this is a good comment - 1st line is 'obvious' from code
but the second line nails it - that is a context that is very valuable - but cannot be read from code